test/Array/toLocaleString.baseline

```diff
@@ -43,3 +43,15 @@ TypeError : The value of the property 'toLocaleString' is not a Function object
 
 9. Array: toLocaleString should use length property
 10.00, 20.00, 30.00, , , 
+
+
+10. Array: toLocaleString should use toLocaleString accessor on element
+accessor-value
+
+
+11. Array: toLocaleString should throw TypeError on undefined
+TypeError : Cannot redefine non-configurable property 'toLocaleString'

```

Comment:
This is a TypeError but not the one I was hoping to trigger. This is happening in the `Object.defineProperty` step, because the previous scenario used `Object.defineProperty` also and didn't specify `configurable: true`, meaning the property can never be reconfigured again.